.github/workflows/extension-attach-artifact-release.yml

```
@@ -155,42 +155,50 @@ jobs:
 
       - name: Run extra command
         if: inputs.extraCommand != ''
+        env:
+          EXTRA_COMMAND: ${{ inputs.extraCommand }}
         run: |
-          ${{ inputs.extraCommand }}
+          ${EXTRA_COMMAND}
 
       - name: Run extra Linux command
         if: inputs.extraLinuxCommand != ''
```

actionlint (1.7.9)

property "extralinuxcommand" is not defined in object type {artifactid: string; artifactpath: string; combinejars: bool; dry_run: bool; dry_run_version: string; extracommand: string; extramavenargs: string; javabuildversion: string; mavenprofiles: string; os: string; zip: string}

(expression)